src/ui/components/Dashboard/DashboardLayout.jsx

```diff
@@ -32,7 +33,7 @@ function DashboardLayout() {
         label: `Notifications ${newNotifs ? `(${unviewedNotifs.length})` : ''}`,
         component: <Notifications notifs={unviewedNotifs} />
       },
-      { key: 'config', label: 'Config', component: <Collections selectedCollection="config" /> },
+      { key: 'config', label: 'Config', component: <Config config="config" /> },
```

Comment:
it is certainly fun to have 1 line with the word "config" in it 5 times, but the component takes no props so we should delete those last 2: `config="config"`